.github/workflows/deploy-build-from-pr.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       WORKFLOW_NAME: "${{ github.event.workflow.name }}"
       WORKFLOW_STATUS: "${{ github.event.workflow_run.conclusion }}"
     steps:
-      - uses: hmarr/debug-action@1201a20fc9d278ddddd5f0f46922d06513892491 # pin@v2
+      - uses: hmarr/debug-action@a701ed95a46e6f2fb0df25e1a558c16356fae35a # pin@v2.1.0
```

Comment:
This seems to be an action only used for debugging. It should thus probably not be in production code. If this is really the only occurrence, could we just remove it entirely instead please?

Reply:
This is the only occurrence in the Magma repo and has been in this file since its original commit. The action only prints the environment variables and therefore may be removed in production code.